cooperator/models/subscription_request.py

```diff
@@ -256,10 +257,11 @@ def _compute_subscription_amount(self):
     share_product_id = fields.Many2one(
         "product.product",
         string="Share type",
-        domain=[("is_share", "=", True)],
+        domain="[('is_share', '=', True), ('company_id', 'in', (company_id, False))]",
```

Comment:
Not sure, but maybe the `company_id` domain is already contemplated in the `ir.rule` of the product module.

Usually, the `company_id` in a multi-company environment is contemplated by `ir.rule` to remove the need to call it explicit in all domains.

Reply:
indeed, as @carmenbianca says, this is to limit the selectable shares in the form in case a user with access to multiple companies want to create a subscription for a other company than the current one. sorry for the confusion, i should have added a comment for this. a comment has now been added.